src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java

```diff
@@ -79,6 +79,13 @@
   public static final String LOGS_DIR_DEFAULT = "logs";
   public static final String LOGS_DIR_DISPLAY = "Logs directory";
 
+  public static final String AVRO_CODEC_CONFIG = "avro.codec";
+  public static final String AVRO_CODEC_DEFAULT = "null";
+  public static final String AVRO_CODEC_DISPLAY = "Avro Compression Codec";
+  public static final String AVRO_CODEC_DOC = "The Avro compression codec to be used for "
+      + "output files. Available values: null, deflate, "
```

Comment:
nit: Line width limit is 100. Maybe linebreaks need adjustment here (still 3 lines probably)

Reply:
The lines here are at most 91 characters long. I think we should be fine with respect to the 100 line limit?